src/sponge/pad.rs

```diff
@@ -40,10 +42,13 @@ mod tests {
         let three = Scalar::from(3u64);
         let four = Scalar::from(4u64);
 
-        assert_eq!(&pad(&[two], 3), &[zero, two, one]);
-        assert_eq!(&pad(&[two, three], 3), &[zero, two, three, zero, one, zero]);
+        assert_eq!(&pad(&[two], 3, zero, one), &[zero, two, one]);
```

Comment:
I would also rename in the tests `zero` and `one` here as `pad_value` and `eom_value` when are used for such purpose, it would be clearer.